base/libgit2/oid.jl

```diff
@@ -1,5 +1,11 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+    GitHash(ptr::Ptr{UInt8})
+
+Construct a `GitHash` from a pointer to `UInt8` data containing the bytes of the
+SHA-1 hash.
```

Comment:
Do we want to mention what happens if you pass a NULL ptr?

Reply:
Good idea, thanks!